vuepress/docs/next/docs/consume/custom-resources.md

```diff
@@ -2,445 +2,54 @@
-EntandoAppPluginLink and creates a path for the Plugin on the Ingress that exposes the EntandoApp in question. This path
-is determined by the `spec.ingressPath` property on the EntandoPlugin custom resource itself. If the EntandoPlugin
-resides in a namespace other than the namespace of the EntandoApp, the EntandoOperator creates a Kubernetes
-Service in the namespace of the EntandoApp that simply delegates to the Service in the namespace of the EntandoPlugin.
-
-### Overview
-
-* Kubernetes Controller Details:
-  * Docker image: [entando/entando-k8s-app-plugin-link-controller](https://hub.docker.com/r/entando/entando-k8s-app-plugin-link-controller) 
-  * Github Repo: [entando/entando-k8s-app-plugin-link-controller](https://github.com/entando-k8s/entando-k8s-app-plugin-link-controller) 
-### Example
-```
----
-kind: "EntandoAppPluginLink"
-apiVersion: "entando.org/v1"
-metadata:
-  name: "test-link"
-  namespace: "your-namespace"
-spec:
-  entandoAppName: your-app
-  entandoAppNamespace: your-namespace
-  entandoPluginName: your-app
-  entandoPluginNamespace: your-namespace
-
-```
-
-### Explanation of properties
-* `spec.entandoAppName` specifies the name of the EntandoApp that requires the plugin, found in `EntandoApp.metadata.name`
-* `spec.entandoAppNamespace` specifies the namespace of the EntandoApp that requires the plugin, found in `EntandoApp.metadata.namespace`
-* `spec.entandoPluginName` specifies the name of the EntandoApp that requires the plugin, found in `EntandoPlugin.metadata.name`
-* `spec.entandoPluginNamespace` specifies the namespace of the EntandoApp that requires the plugin, found in `EntandoPlugin.metadata.namespace`
-
-## EntandoDatabaseService
-
-By default, the Entando Operator deploys one of either the MySQL or PostgreSQL database Docker images for every custom
-resource that requires a database. Many customers may, however, have existing infrastructure for their databases which
-they may want to leverage. The EntandoDatabaseService custom resource allows customers to deploy a Service that points
-to an external database. When deploying one of the Entando Custom Resources that require a database, the Entando
-Operator will look for EntandoDatabases in the same namespace. If it finds one with the same `spec.dbms` setting 
-as the database required by the custom resource, it will create a dedicated schema/username/password combination
-for the custom resource and point the deployment emanating from Custom Resources to this external database.   
-
-
-### Example
-```
----
-kind: "EntandoDatabaseService"
-apiVersion: "entando.org/v1"
-metadata:
-  name: "test-database-service"
-  namespace: "your-namespace"
-spec:
-  dbms: "oracle"
-  host: 10.0.12.41
-  port: 1521
-  databaseName: yourdb 
-  tablespace: 
-  secretName: some-secret
-  jdbcParameters:
-    maxStatements: 300 
-    loginTimeout: 180
-```
+## Entando Core Custom Resource
```

Comment:
1. This should be plural. There are a set of standard or core custom resources involved in a basic installation, not just 
one.
2. As of 7.0, the operator only observes a single namespace.

I'm not sure I have it 100% correct but maybe this:
"The core set of resources are required for the basic installation of Entando in a Kubernetes cluster. The Entando Operator observes these resources in the namespace. If the core custom resource is created, updated or deleted, the operator will trigger a new run-to-completion pod that implements the state change in the cluster. Typically, this results in the deployment of Docker images, services, and sometimes an ingress. Docker images that implement these run-to-completion pods are referred to as Entando Kubernetes Controllers."